README.md

```diff
@@ -4,51 +4,62 @@
 
 > Snitch is inline-defs on steroids
 
+Or:
 
-Snitch is how I debug and understand data flowing through my system.
-It's not a replacement for a full-fledged debugger, but it's pretty close and will be useful in 90% of the cases. 
-I use it as my sole debugger (in rare cases I reach for print statements). 
+> Highly enhanced ergonomics for repl-driven development.
 
-# Features 
-- Support for Clojurescript. 
-- Editor agnostic (use it along with cider, conjure, calva, or cursive!).
-- Highly ergonomic for repl-driven development.
+Snitch is how I debug and understand data flowing through my system. I use it as my sole debugger (in rare cases I reach for print statements).
 
-# What people have to say ❤️ 
+It's not a replacement for a full-fledged debugger, but it's pretty close and will be useful in 90% of the cases. However, unlike a debugger Snitch runs while your application us running, which is both better and worse than a traditional debugger. Better because a debugger freezes your application while you inspect the state. Better because the Clojure REPL will let you modify the code you are debugging, while it is running. Most debugger do not let you do that. Worse because Snitch will only show you the latest value from a series of invocation, where a debugger will let you step to the invocation you are interested in. (BTW. There are ways to collect state from all invocations, even if Snitch doesn't help with it.)
+
+# Features
+- `defn*`, `*let`, `*fn`, and `defmethod*` macros automating inline `def`s for function arguments and bindings
+- Gives access to the return value of called functions
+- Lets you modify and then rerun the function with the snitched invokation arguments
+- Support for Clojurescript.
+- Editor agnostic (use it along with [CIDER](https://cider.mx), [Conjure](https://conjure.oli.me.uk/), [Calva](https://calva.io), [Cursive](https://cursive-ide.com/), or any REPL enhanced [Clojure editor](https://clojure.org/guides/editors!).
+
+# What people have to say ❤️
 > Very handy with those variants of the regular macros. Just add a `*` and you are inline def-ing the args! - Peter Strömberg (co-author of Calva)
 
 # Talk
-I gave a [talk](https://www.youtube.com/watch?v=WqilQulsJQc) about snitch at clojure Asia you can watch it 
+I gave a [talk](https://www.youtube.com/watch?v=WqilQulsJQc) about snitch at clojure Asia you can watch it
 here https://www.youtube.com/watch?v=WqilQulsJQc.
 
 # Usage
 ## Installation
-I recommend adding snitch to your `~/.lein/profiles.clj`.
+I recommend adding snitch to your `~/.lein/profiles.clj` or `deps.edn`.
```

Comment:
I don't personally use deps.edn so I don't know much about it, is there some version of profiles.clj for deps? maybe I can add that here.

Reply:
The version of project.clj for deps is called `deps.edn` 😄. Actually I don't think you need to show either. Linking to the Clojars page lists the coordinates for both.

Reply:
No, what I meant was that `~/.lein/profiles.clj` adds it to your personal dependencies without adding it to a project's dependencies, is there something for deps.edn that is like that?